cmd/repo-init/api.go

```diff
@@ -604,6 +629,14 @@ func (s server) generateConfig(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	s.logger.Debug("running 'make jobs' prior to commit")
+	makeJobs := exec.Command("make", "jobs")
```

Comment:
Will this work though when we run this inside a Pod? `make jobs` runs `podman run`...

Reply:
No, I suppose it will not work...I wasn't actually aware of that.
I think this means the only way to achieve this would be a refactor of: `ci-operator-prowgen`, `sanitize-prow-jobs`, and `ci-operator-checkconfig` to allow us to directly use them within the `repo-init` binary. Do you see any way around that?

Reply:
Or do we simply need to include `podman` in the dockerfile for `repo-init`? not sure if that would solve the issue...

Reply:
Or does that just mean we need to include `podman` in the `repo-init` dockerfile?

Reply:
I went ahead and added `podman` to the dockerfile. If we need to go the other route I am going to do an entirely new PR and close this one.

Reply:
I never knew kube/ocp well enough to know whether you can successfuly run `podman run` in containers ;) If it does not work, we could do what `autoconfigbrancher` does:

1. Inject the necessary binaries into the `repo-init-ui` image ([link](https://github.com/openshift/release/blob/d1118ae65699cab930f93c741121bdd1c2305874/ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml#L182-L210))
2. Execute the binaries themselves ([link](https://github.com/openshift/ci-tools/blob/3a9d6085e8ea08c322794d9e27084e813c52e647/cmd/autoconfigbrancher/main.go#L197-L203))